book/applicaster-video-platform/js2native/readme.md

```diff
@@ -143,69 +143,54 @@ TVShowID: "1",
 ```
  #### Morpheus.updateUserProfile()
  Update user profile information. The old properties will be overwritten by the new ones.
+ 
  **userProperties** - The user properties information. The keys of the
-JSON object should be formatted as pascal case (*PascalCase*). Morevoer,
-properties which are not *custom properties* are called *special properties*.
-They are not required, but if delivered, should be written as follows:
- * Name
+JSON object should be formatted as pascal case (*PascalCase*). 
+Moreover, properties which are not *custom properties* are called *special properties*. They are not required, but if delivered, should be written in the following format: "CustomFieldName".
+
+The following is an example of a javascript code to send user data:
+```javascript
+function sendData(){
+	var userProperties = 
+		"timestamp": 1458061391636,
+		"Name": "Neo",
+		"FirstName": "Thomas",
+		"LastName": "Anderson",
+		"Email": "t.anderson@metacortex.com",
+		"Phone": "1",
+		"UserName": "UserName",
+		"AnyCustomField1": "1",
+		"AnyCustomField2": "2",
+		"AnyCustomField3": "3",
+		"SocialIDs": {
+			"facebook": {
+					"ID": "123"
+			},
+			"twitter": {
+				"ID": "123"
+			},
+			"google": {
+				"ID": "123"
+			}
+}
+ js2n.Morpheus.updateUserProfile(userProperties);
+```
+ * Note that in the example above we have field called SocialIDs. If you have any sort of social network login, please make sure to follow the structure there, rather than send the IDs as a separate custom values. This enables us to search for Social IDs properly and for you to easily add new social providers without a change to the structure.
```

Comment:
```suggestion
 * Note that in the example above we have a field called SocialIDs. If you have any sort of social network login, please make sure to follow the structure there, rather than send the IDs as separate custom values. This enables us to search for Social IDs properly and for you to easily add new social providers without a change to the structure.
```